src/locales/en-US/commands.json

```diff
@@ -726,8 +726,23 @@
     "clickHere": "**[Click here to join my support server](https://invite.switchblade.xyz)**"
   },
   "presidentialalert": {
-    "commandDescription": "Generates a Presidential Alert notifications",
+    "commandDescription": "Generates a Presidential Alert notification",
     "commandUsage": "<text>",
     "missingText": "You need to give me some text for the alert"
+  },
+  "tictactoe": {
+    "commandDescription": "Challenge someone to a game of Tic Tac Toe",
+    "commandUsage": "<user to challenge>",
+    "missingUser": "You need to give me someone to challenge",
+    "title": "Tic Tac Toe",
+    "pleaseWait": "Please wait, loading board...",
+    "currentPlayer": "Current player",
+    "itsYourTurn": "It's your turn, {{player}}",
+    "itsATie": "It's a tie!",
+    "wins": "{{player}} wins!",
+    "timeout": "This Tic Tac Toe invite has timed out.",
+    "hasChallenged": "{{player}} has challenged you to a game of Tic Tac Toe",
+    "clickTheReaction": "Click the {{CONFIRMATION_EMOJI}} reaction button below to accept.",
+    "thisTimeoutsIn": "This timeouts in {{COLLECTOR_TIMEOUT}} seconds"
```

Comment:
I think this would be better as „This **will timeout** in {{COLLECTOR_TIMEOUT}} seconds”, but I'll wait for @Lireoy's opinion.

Reply:
I'm not sure. I considered something like: "Respond in {{COLLECTOR_TIMEOUT}} seconds."

Reply:
"Answer" should in theory be better

Reply:
I'm with @LogGame-HU-DEaN on this one.

Reply:
I think it's more user friendly something like `You have {{COLLECTOR_TIMEOUT}} to accept it`.

Reply:
Agree. What do you guys think?

Reply:
###### use reactions to agree with suggestions pls

Reply:
I think we can change it. 4 upvotes on Davi's suggestion

Reply:
5 now